src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -35,6 +35,14 @@ A recommended approach.
 * [You might not need Redux: The Flutter edition][], by Ryan Edge
 * [Making sense of all those Flutter Providers][]
 
+## Riverpod
+
+A provider rewrite by the same author, compiled safe, more testable, more flexible
+
+* [Riverpod Homepage][]
+* [Riverpod Github][]
+* [Motivation for provider rewrite][]
+
```

Comment:
Please put this section at the end of the page. We might move things around at some point but right now we're adding approaches in "first come, first serve" order.